<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerPolyDataCompressedTransmissionModuleWidget</class>
 <widget class="QWidget" name="qSlicerPolyDataCompressedTransmissionModuleWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>814</width>
    <height>797</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="ConnectorPropertyFrame">
     <property name="title">
      <string/>
     </property>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>202</x>
        <y>52</y>
        <width>424</width>
        <height>123</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="ConnectorHostAddressLabel">
           <property name="text">
            <string>Host name:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="ConnectorHostAddressEdit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>0</height>
            </size>
           </property>
           <property name="inputMethodHints">
            <set>Qt::ImhDigitsOnly</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="ConnectorPortLabel">
           <property name="text">
            <string>  Port:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="ConnectorPortEdit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>0</height>
            </size>
           </property>
           <property name="inputMethodHints">
            <set>Qt::ImhDigitsOnly</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="FrequencyLabel">
             <property name="text">
              <string>Frequency:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="FrameFrequency">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>0</height>
              </size>
             </property>
             <property name="inputMethodHints">
              <set>Qt::ImhDigitsOnly</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLabel" name="UseCompressLabel">
             <property name="text">
              <string>Use Compress:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="UseCompressCheckBox">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Active</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="ConnectorStateCheckBox">
             <property name="minimumSize">
              <size>
               <width>19</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Start Video</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="StartVideoCheckBox">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
 </customwidgets>
 <tabstops>
  <tabstop>ConnectorStateCheckBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
